cookbooks/redis/recipes/install.rb

```diff
@@ -17,6 +17,12 @@
 
 run_installer = !FileTest.exists?(redis_base_directory) || node['redis']['force_upgrade']
 
+# check if redis-server exists
```

Comment:
Move the block in https://github.com/engineyard/ey-cookbooks-stable-v6/pull/100/files#diff-282ec2fdc23e486143f8455f9feab8d2R104 up and use the `redis_bin_path` in the checks here.

Reply:
Made the change.